### PR TITLE
Fix errors of commit  "Optimize Fragment Shader Stage Output with pac…

### PR DESCRIPTION
**Pull request by jiaolu**
…k intrincis" when compCount is not even

### CODE REVIEW

File: context/llpcContext.h [hunk 1]
```diff
@@ -292,6 +294,8 @@ class Context : public llvm::LLVMContext
         llvm::Type* pDoubleTy;    // Double
         llvm::Type* pVoidTy;      // Void
 
+        llvm::Type* pInt16x2Ty;   // Int16 x 2
+        llvm::Type* pInt16x4Ty;   // Int16 x 4
```
Comment (linqun) [inline context/llpcContext.h]: It seems this type isn't used, please remove it.

**Comment by amdrexu** [comment]
Resolve the issue and update the commit.

File: context/llpcContext.h (hunk 1 above)
Comment (jiaolu) [reply context/llpcContext.h]: commit updated

**Comment by JacobHeAMD** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 025c1612fb8d91b5a2de4555e26dc9756032cb90
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev b563d58ad6dc7c2daee9b73801dcc1321686b73b</li>
<li>SPVGEN: dev 5378c7618a2a3bf60ba01a86db980930e8581146</li>
<li>XGL: dev 6aa9683c02873844f932a920bdb4f688dedec724</li>
<li>LLVM: amd-vulkan-dev 1ca248e6fc6d4d15491756f92f4da139222ef3ca</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 901/901 (100.0%)</li>
<li>Failed: 0/901 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.1.2-cherry-picks)</summary><ul>


<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 312622/312622 (100.0%)</li>
<li>Failed: 0/312622 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 312622/312622 (100.0%)</li>
<li>Failed: 0/312622 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**